docs/framework/react/reference/functions/useCreateAtom.md

```
@@ -0,0 +1,86 @@
+
+### Returns
+
+`ReadonlyAtom`\<`T`\>
+
+### Example
+
+```tsx
+const countAtom = useCreateAtom(0)
+```
+
+## Call Signature
```

markdownlint-cli2 (0.22.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)